packages/ui/src/components/molecules/Prefabs/EditButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Button, Typography } from '../../atoms';
+
+export interface EditButtonProps {
+  text?: string;
+  onClick?: () => void;
```

Comment:
text prop should be required

```suggestion
  text: string;
  onClick?: () => void;
```

Reply:
updated